Vagrantfile

```diff
@@ -1,11 +1,14 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
+load "settings.sh"
+
+$IP = NETWORK + ".18".to_s
```

Comment:
- Do you need `.to_s` here?
 - Do you need `$`?
 - Why do you hard-code the number `.18`?
 - Why don't you use `LAN_IP`?